### PR TITLE
configurable battery details

### PR DESCRIPTION
**Pull request by HannesBachter**

### CODE REVIEW

**Review by floweisshardt — CHANGES_REQUESTED** [review]
Do not merge. This will break with cob4 base compatibility. 

Maybe those changes are not needed at all. See https://github.com/mojin-robotics/cob4/issues/2526#issuecomment-1209493456

File: cob_bms_driver/src/fake_bms.py [hunk 1]
```diff
@@ -25,66 +25,66 @@
...
-        self.voltage              = 48.0
-        self.current              = -8.0
-        self.remaining_capacity   = 35.0
-        self.full_charge_capacity = 35.0 # Ah
-        self.temperature          = 25.0
+        self._voltage              = 48.0 # V
+        self._current              = -6.0 # A
+        self._remaining_capacity   = 60.0 # Ah
+        self._full_charge_capacity = 60.0 # Ah
+        self._temperature          = 25.0 # °C
```
Comment (fmessmer) [inline cob_bms_driver/src/fake_bms.py]: I would make those configurable via a new `battery.yaml` in `mojin_hardware_config`, i.e. `rospy.get_param('voltage', 48)` defaulting to the old/cob4 values...i.e. open a PR that adds these `battery.yaml` file in `mojin_robots` and add a `rosparam` line in the respective `mojin_bringup` launch files

**Review by fmessmer — CHANGES_REQUESTED** [review]

File: cob_bms_driver/src/fake_bms.py (hunk 1 above)
Comment (fmessmer) [reply cob_bms_driver/src/fake_bms.py]: make sure the `battery.yaml` includes also the values for https://github.com/ipa320/cob_docking/pull/84#discussion_r941630133

**Review by fmessmer — APPROVED** [review]

File: cob_bms_driver/src/fake_bms.py (hunk 1 above)
Comment (fmessmer) [reply cob_bms_driver/src/fake_bms.py]: done in https://github.com/mojin-robotics/mojin_robots/pull/515

**Pull request merged**